support-frontend/assets/helpers/tracking/acquisitions.ts

```diff
@@ -150,9 +150,7 @@ function buildReferrerAcquisitionData(
 	const campaignCode =
 		getCampaignCode() ??
 		(acquisitionData.campaignCode as string | undefined) ??
-		getQueryParameter('INTCMP') ??
-		getQueryParameter('CMP') ??
-		undefined;
+		(getQueryParameter('INTCMP') || getQueryParameter('CMP') || undefined);
```

Comment:
A bunch of lint errors in this file initially blocked my merge commit from `main`, so I fixed them- this isn't otherwise related to the DS landing page. I think this was an instance where we were previously using logical OR (`||`) as _both_ a nullish coalescer (`??`) to deal with `null` or `undefined`, _and_ to circumvent falsey values that are _not_ `null` or `undefined`.

`getQueryParameter` will always return a string, but if it doesn't find a value for the passed parameter key name it will return an empty string. The nullish coalescing operator thinks an empty string is fine so will resolve to it and ignore the right hand side of the expression. However in this case if `'INTCMP'` returns empty string we want to look for `'CMP'` instead, so we want logical OR, which will resolve to the right hand side of the expression if the left hand side is any kind of falsey value.